infinite_tracing/lib/infinite_tracing/transformer.rb

```diff
@@ -39,14 +39,14 @@ def safe_param_name value
       end
 
       def hash_to_attributes values
-        values.map do |key, value|
+        values.to_h do |key, value|
```

Comment:
I didn't know this syntax was possible! nice!

Reply:
Neither did Ruby 2.5.9 apparently. 😞 Reverted and left a TODO in `.rubocop.yml` with 8ad6c42